docs/client/introduction.mdx

```diff
@@ -45,12 +45,10 @@ As shown in the diagram below, implementing an experiment using a Statsig client
 
 :::info Best Practices
 
-**Using [**StatsigUser**](/client/concepts/user)**
-
+**Using [StatsigUser](/client/concepts/user)**
 Learn how to use [StatsigUser](/server/concepts/user) while using a client SDK. 
```

Comment:
the header here links to client/concepts/user, and then the body links to server/concepts/user - it should be the client one, but Id just simplify it down to one link